nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/vendor/varian/VNMRPar.java

```diff
@@ -115,10 +115,12 @@ public static LinkedHashMap<String, String> getParMap(String fpath) {
         try {
             String handle = processVNMRParFile(fpath);
             ArrayList<String> parNames = getPars(handle);
-            hmap = new LinkedHashMap();
+            hmap = new LinkedHashMap<>();
             for (String parName : parNames) {
                 VNMRPar vp = getPar(handle, parName);
-                hmap.put(parName, vp.getJValues());
+                if (vp != null) {
+                    hmap.put(parName, vp.getJValues());
+                }
```

Comment:
Changed this to how possible NPE were handled in other overloaded version of function: https://github.com/nanalysis/nmrfx/blob/4145ec117ba89f6926ff2be1dc000cdcaaa25900/nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/vendor/varian/VNMRPar.java#L145